packages/components/menu/src/menuItem.vue

```diff
@@ -50,6 +50,7 @@ import {
 import ElTooltip from '@element-plus/components/tooltip'
 import { RootMenuProvider, SubMenuProvider } from './menu'
 import useMenu from './useMenu'
+import { Effect } from 'element-plus'
```

Comment:
Even though this is legit. But I think we better use `@element-plus/component/popper`